Godeps/_workspace/src/github.com/ipfs/go-datastore/measure/measure.go

```diff
@@ -179,10 +179,9 @@ func (m *measure) Batch() (datastore.Batch, error) {
 func (mt *measuredBatch) Put(key datastore.Key, val interface{}) error {
 	mt.puts++
 	valb, ok := val.([]byte)
-	if !ok {
-		return datastore.ErrInvalidType
+	if ok {
+		_ = mt.m.putSize.RecordValue(int64(len(valb)))
```

Comment:
@whyrusleeping I somewhat agree, but this change will touch a lot of code and change the datastore interface.  Since it doesn't seam necessary for getting my changes in can we agree to hold of on this until this pull request is ready to merge?